sycl/source/detail/graph_impl.cpp

```diff
@@ -1893,6 +1893,13 @@ void executable_command_graph::update(const std::vector<node> &Nodes) {
   impl->update(NodeImpls);
 }
 
+dynamic_parameter_base::dynamic_parameter_base(
+    command_graph<graph_state::modifiable> Graph, const property_list &PropList)
+    : impl(std::make_shared<dynamic_parameter_impl>(
+          sycl::detail::getSyclObjImpl(Graph))) {
+  checkGraphPropertiesAndThrow(PropList);
```

Comment:
Is this required for dynamic parameters? This checks for graph properties but any properties passed here should be dynamic_parameter properties (if/when they exist).